dns_sd_bonjour.c

```diff
@@ -78,12 +78,12 @@ static void __cfnet_browser_cb (
 		IIO_DEBUG("DNS SD: Net service is null.\n");
 		goto verify_flags;
 	}
-	
+
```

Comment:
whitespace change could be it's own commit

Reply:
well, that is my editor doing this automatically. If I have to come with a v3 I will make this changes in vim so I don't get this automatically done